react-hash-routing/client/components/header.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function Header(props) {
+  return (
+    <header className="mb-5">
+      <nav className="navbar navbar-dark bg-dark shadow-sm">
+        <div className="container">
+          <div className="col px-0">
+            {/* this anchor should go back to the catalog at '#' */}
+            <a href="" className="navbar-brand">
```

Comment:
While this works to go back home it is better to set the `href` to a `#`: `href="#"`. It shows intention and doesn't look like you just forgot to put something in there.